README.md

```diff
@@ -42,20 +43,27 @@
     - [See here for more information about publishing packages](#see-here-for-more-information-about-publishing-packages)
     - [See the top answer on this stackoverflow question for why you may want to build against multiple node versions. Probably unnecessary for this workflow.](#see-the-top-answer-on-this-stackoverflow-question-for-why-you-may-want-to-build-against-multiple-node-versions-probably-unnecessary-for-this-workflow)
 
-## Quickstart Guide
+## Pre-requisites:
 
-Using the Unity Bootstrap Theme and React components for building ASU Web Standards 2.0 compliant web sites and apps.
+- Node.js (v20.15.1 or higher)
+- Yarn (v4)
+
+This monorepo uses Yarn 4 Workspaces to manage dependencies and build packages. This means that you should use Yarn to install and manage dependencies, and to run scripts. If you don't have Yarn installed, you can install it by running ```npm install -g yarn```.
```

Comment:
We can remove this line, and replace it with:
This workflow uses Volta version 1.1.1. Install that version using
```
sh getvolta/getvolta.sh --version 1.1.1
```

Reply:
Let's also add, "Verify that a yarnrc.yml file is not in your home folder. Yarn will use that as default instead of the project's version of that file."